src/Google.Cloud.Functions.Templates/templates/event-function-cs/Function.cs

```diff
@@ -1,17 +1,3 @@
-// Copyright 2020, Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
```

Comment:
super nit for the future: Maybe name all CloudFunction as the vb one. This is really not important, just one of those things...

Reply:
I'd *prefer* not to - I wish I could use Function for VB as well, and that's what I tried at first. I only made that CloudFunction because Function is a keyword in VB :)